src/cpp/tests/testthat/run-tests.R

```diff
@@ -0,0 +1,17 @@
+library(testthat)
+
+runAllTests <- function(sourceDir, outputDir, filter = NA)
+{
+   testThatDir <- file.path(sourceDir, "tests/testthat")
+   
+   if (!is.na(filter))
+   {
+      tests <- testthat::test_dir(testThatDir, filter = filter)
+   } 
+   else
+   {
+      tests <- testthat::test_dir(testThatDir)
+   }
+   
+   cat(sum(as.data.frame(tests)$failed), file = file(file.path(outputDir, "testthat-failures.log")))
```

Comment:
You can remove the file() call (just give the path to the file, no need to open a connection)

Reply:
Missed that, thanks! I meant to just move the file.path part.